src/txn/txn_rollback_to_stable.c

```diff
@@ -1158,12 +1218,14 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
     WT_TXN_GLOBAL *txn_global;
     wt_timestamp_t max_durable_ts, newest_start_durable_ts, newest_stop_durable_ts,
       rollback_timestamp;
+    uint64_t rollback_txnid;
     size_t addr_size;
     char ts_string[2][WT_TS_INT_STRING_SIZE];
     const char *config, *uri;
-    bool durable_ts_found, prepared_updates;
+    bool durable_ts_found, prepared_updates, rollback_recovery_txnid;
```

Comment:
Instead of `rollback_recovery_txnid` flag, how about `has_txn_updates_gt_than_ckpt_snap`.

Reply:
Fixed.